webapp/channels/src/components/widgets/modals/components/base_setting_item.tsx

```diff
@@ -57,19 +58,17 @@ function BaseSettingItem({title, description, content, className, error, descrip
         </div>
     );
 
-    const getClassName = classNames('mm-modal-generic-section-item', className);
-
     return (
-        <div className={getClassName}>
-            {Title}
-            {descriptionAboveContent ? Description : undefined}
+        <div className={classNames('mm-modal-generic-section-item', className)}>
+            {titleComponent}
+            {descriptionAboveContent ? descriptionComponent : undefined}
             <div
                 data-testid='mm-modal-generic-section-item__content'
                 className='mm-modal-generic-section-item__content'
             >
                 {content}
             </div>
-            {descriptionAboveContent ? undefined : Description}
+            {descriptionAboveContent ? undefined : descriptionComponent}
```

Comment:
Is this supposed to be the other way around like the above? Or should they be different?

Reply:
yes its supposed to be other way round